docs/hub/security-gpg.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Signing commits with GPG
+---
+
+<h1>Signing commits with GPG</h1>
+
+`git` has an authentication layer to control who can push commits to a repo, but it does not authentify the actual commit authors.
+
+In other words, you can commit changes as `Elon Musk <elon@tesla.com>`, push them to your preferred `git` host (for instance github.com), and your commit will link to Elon's GitHub profile. (Try it! But don't blame us if Elon gets mad at you for impersonating him.)
+
+See this post by Ale Segala for more context: [How (and why) to sign `git` commits](https://withblue.ink/2020/05/17/how-and-why-to-sign-git-commits.html)
+
+You can prove a commit was authored by you, using GNU Privacy Guard (GPG) and a key server. GPG is a cryptographic tool used to verify the authenticity of a message's origin. We'll explain how to set this up on hf.co below.
+
+The Pro Git book is, as usual, a good resource about commit signing: [Pro Git: Signing your work](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work).
+
+
+## Why we implemented signed commits verification on the Hugging Face hub
+
+The reasons we implemented GPG signing were:
+- To provide finer-grained security, especially as more and more Enterprise users rely on the Hub (in the future you'll be able to disallow non-signed commits)
```

Comment:
I would advise against talking about future features in the documentation (this is probably better in a blog post) since it creates expectations for users and if we don't deliver on them, users may be disappointed. The docs also reflect the current state of the product, and if it doesn't exist yet, then I don't think we need to mention it :)

```suggestion
- To provide finer-grained security, especially as more and more Enterprise users rely on the Hub.
```